packages/connect-react-query/src/method-unary-descriptor.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021-2023 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import type { Message, MethodInfoUnary } from "@bufbuild/protobuf";
+
+/** Defines a standalone method and associated service  */
+export type MethodUnaryDescriptor<
+  I extends Message<I>,
+  O extends Message<O>,
+> = MethodInfoUnary<I, O> & {
+  localName: string;
+  service: {
+    typeName: string;
+  };
```

Comment:
```suggestion
  readonly service: Omit<ServiceType, "methods">;
```
Because we may want other features like options in the future.